documentation/modules/ROOT/pages/integrations/outbox.adoc

```diff
@@ -124,6 +124,55 @@ The {prodname} CDC connector in conjunction with the Outbox Event Router will be
 
 To see a full end-to-end demo, the https://github.com/debezium/debezium-examples/tree/main/outbox[Outbox] example illustrates two Quarkus microservice applications using the outbox pattern to share data between them when orders are placed or cancelled.
 
+[id=reactive-outbox]
+== Reactive Variant
+
+If your application is making use of Reactive Datasources or Hibernate Reactive, then there are some differences. 
+
+To import the reactive variant, the extension is added as follows:
+
+[source,xml,subs="verbatim,attributes"]
+----
+<dependency>
+  <groupId>io.debezium</groupId>
+  <artifactId>debezium-quarkus-outbox-reactive</artifactId>
+  <version>{debezium-version}</version>
+</dependency>
+----
+
+The extension provides the application with the `io.debezium.outbox.quarkus.ExportedEvent` interface just like in the non reactive variant, but also comes with the `DebeziumOutboxHandler` class which can be injected into your applications and exposes the `persistToOutbox` method. One example invocation is seen below which is using the same `ExportedEvent` as the previous example:
```

Comment:
```suggestion
The reactive extension provides the application with the same `io.debezium.outbox.quarkus.ExportedEvent` interface as the non-reactive variant, but it also provides the `DebeziumOutboxHandler` class.
This class can be injected into your applications and expose the `persistToOutbox` method. 
The following example shows an invocation that uses the same `ExportedEvent` as in the earlier example:
```

Reply:
In effect, @roldanbob, there is a lot of "wiring" required when you're working with reactive modules, and so inside Quarkus you can inject the `DebeziumOutboxHandler` bean into your application, and it provides you directly with a method that allows you to quickly persist the `ExportedEvent` to the outbox table without needing to be concerned about how to wire all the dependencies required to do this.

Reply:
I made a slight change to my original edit that incorporates information from your comment to improve the clarity of the paragraph.